samples/python/multicam_image_selector.py

```diff
@@ -0,0 +1,1156 @@
+    ap.add_argument("--square", type=float, default=0.0, help="Square size (meters) for Charuco/GridBoard")
+    ap.add_argument("--marker", type=float, default=0.0, help="Marker size (meters) for Charuco/GridBoard")
+    ap.add_argument("--separation", type=float, default=0.0, help="Marker separation (meters) for GridBoard")
+    ap.add_argument("--gridboard", type=str, help="GridBoard WxH (e.g., 7x5). Only used for --pattern aruco")
+    ap.add_argument(
+        "--expected-aruco-markers",
+        type=int,
+        default=0,
+        help="Expected number of ArUco markers (for scoring normalisation). 0 = default",
+    )
+    ap.add_argument("--per-camera", type=int, default=80, help="Target images per camera")
+    ap.add_argument(
+        "--selector",
+        choices=["kmeans", "greedy", "random"],
+        default="kmeans",
+        help="Selection algorithm. Note: random is quality-aware (top scores first).",
+    )
+    ap.add_argument(
+        "--kmeans-standardize",
+        action="store_true",
+        help="Standardize features before k-means/greedy",
+    )
+    ap.add_argument(
+        "--greedy-min-dist",
+        type=float,
+        default=0.6,
+        help="Min feature-space distance for greedy diversity",
+    )
+    ap.add_argument("--seed", type=int, default=123, help="Random seed")
+    ap.add_argument(
+        "--require-all-cams",
+        action="store_true",
+        help="Select frames where all cams detect the pattern (by frame id)",
+    )
+    ap.add_argument(
+        "--strict-all-cams",
+        action="store_true",
+        help="With --require-all-cams, error out if no common frames exist",
+    )
+    ap.add_argument("--jobs", type=int, default=0, help="Parallel workers (0=auto, 1=single)")
+    ap.add_argument("--max-size", type=int, default=0, help="Max long side in px for processing (0 = full res)")
+    ap.add_argument("--fix-orientation", action="store_true", help="Autorotate using EXIF (requires Pillow)")
+    ap.add_argument("--min-sharpness", type=float, default=0.0, help="Minimum area-normalized Laplacian variance")
+    ap.add_argument("--min-corners", type=int, default=0, help="Minimum detected corners/points")
+    ap.add_argument("--w-sharp", type=float, default=0.40, help="Weight for sharpness (clamped >=0)")
+    ap.add_argument("--w-exposure", type=float, default=0.30, help="Weight for exposure sanity (clamped >=0)")
+    ap.add_argument("--w-corners", type=float, default=0.20, help="Weight for corner/marker coverage (clamped >=0)")
+    ap.add_argument("--w-angle", type=float, default=0.10, help="Weight for orientation diversity (clamped >=0)")
+    ap.add_argument("--relative-to", help="Write relative paths in YAMLs with respect to this directory")
+    ap.add_argument(
+        "--yaml-legacy",
+        action="store_true",
+        help="Prepend %%YAML:1.0 for legacy consumers",
+    )
```

Comment:
I would say it should be default behaviour without option. It works with both Python and OpenCV own FileStoreage class.